packages/frontend/src/pages/Editor/routes.tsx

```diff
@@ -10,6 +12,15 @@ export default function EditorRoutes(): React.ReactElement {
       <Route path="/create" element={<CreateFlowPage />} />
 
       <Route path="/:flowId" element={<EditorPage />} />
+
+      <Route
+        path="/:flowId/notifications"
+        element={
+          <EditorSettingsLayout>
```

Comment:
nit: thinking if it's better to abstract this out into a `EditorSettingsPage` component or similar... makes it less messy if we want to add more settings sections in the future